pages/wizard/wizard-form.page.tsx

```diff
@@ -31,7 +31,13 @@ export default function WizardPage() {
       <button id="set-loading" onClick={() => setIsLoading(true)}>
         Set loading
       </button>
-      <Wizard id="wizard" steps={steps} i18nStrings={i18nStrings} isLoadingNextStep={isLoading} />
+      <Wizard
+        id="wizard"
+        steps={steps}
+        i18nStrings={i18nStrings}
+        isLoadingNextStep={isLoading}
+        nextStepLoadingText="Loading next step"
```

Comment:
Why can't this be in i18nStrings if its a static property?

Reply:
It doesn't necessarily need to be static. If people wanted, they could add a different loading message for the last button (the Submit button): `nextStepLoadingText={activeStepIndex === lastStepIndex ? "Submitting form" : "Loading next step"}`.

I also purposefully aligned this with the `isLoadingNextStep` property, since those two will be used together in most cases.